content/docs/04-clusters/02-data-center/01-maas/02-architecture.md

```diff
@@ -22,11 +23,34 @@ The following are some architectural highlights of bare-metal Kubernetes cluster
 - Palette provides a cloud-like experience for deploying clusters on bare metal servers. The result is increased performance at minimal cost and operational effort.
 
 
-- A Private Cloud Gateway (PCG) that you install into a MAAS cloud using a local installer will facilitate communication between Palette and MAAS. This is necessary because Palette needs to communicate with a specific MAAS server. However, MAAS environments are typically in a private network without a central endpoint. The PCG provides this endpoint and also wraps the MAAS environment into a cloud account that you can target for cluster deployment in Palette.  
+- A Private Cloud Gateway (PCG) that you install in a MAAS cloud using a local installer facilitates communication between Palette and MAAS. The PCG is necessary in MAAS environments where Palette does not have direct network access to the MAAS server. Since MAAS environments are typically in a private network without a central endpoint, the PCG provides this endpoint and also wraps the MAAS environment into a cloud account that you can target for cluster deployment in Palette.  
+
+  <br />
+
+  The diagram below illustrates how MAAS works with Palette using a PCG.
 
+  <br />
 
 - When the PCG is installed, it registers itself with a Palette instance and enables secure communication between the SaaS portal and the private cloud environment. The gateway enables installation and end-to-end lifecycle management of Kubernetes clusters in private cloud environments from Palette's SaaS portal.
 
-![Network flow from an architectural perspective of how MAAS works with Palette](/maas_cluster_architecture.png)
 
 
+- Self-hosted Palette instances can communicate directly with the MAAS environment if both resources can access each other directly over the network. In this scenario, you can use the system PCG. When you register a MAAS account with Palette, you would select **Use System Private Gateway** to enable direct communication between Palette and MAAS. Refer to the [Register and Manage MAAS Cloud Account](/clusters/data-center/maas/register-manage-maas-cloud-accounts) guide to learn more. For the self-hosted Palette instance, MAAS is reachable on port 5240.
+
+  
+
+The table below lists when you would use Palette's PCG or the System Private Gateway. 
+
+<br />
+
+| Attribute | PCG | System Private Gateway |
+|-----------|----|----------------|
+| Connects to Palette SaaS. | ✅ | ❌ |
+| Connects to self-hosted Palette. | ✅ | ✅ |
+| Direct network access to MAAS from self-hosted Palette.  |  ❌ | ✅ |
+| Environment requires internet access. |   ✅  | ❌ |
+
```

Comment:
It will also help if you explain each row. This can be as simple as a bullet list or something similar. But from a reader's perspective, it can be difficult to understand the table without some additional context.

Reply:
Added it in the section.